truss/parsesvcname/parsesvcname.go

```diff
@@ -0,0 +1,87 @@
+// Package parsesvcname will parse the service name of a protobuf package. The name returned will always be camelcased.
```

Comment:
Ah so we are getting the service name from the generated go code from the output of `protoc-gen-go`. So we are getting the camel case.

It is important to note that it will also convert a name from `foo_bar` => `FooBar`. As this will change the folder structure.

```
syntax = "proto3"
service foo_bar {
```
Will create folder `/foobar-service/` rather than `foo_bar-service`.

Not really sure where to note this. 

Thoughts on `log.Warn` when the `ToLower` of the service name in the protofile != `ToLower` of the generated go code service name? This could happen later in the pipeline, but just outputting something so that the user knows that the folder name does not have the underscore.